torch/nn/functional.py

```diff
@@ -3262,3 +3263,147 @@ def _in_proj_v(weight, bias, embed_dim, value):
         attn_output_weights = None
 
     return attn_output, attn_output_weights
+
+@weak_script
+def transformer_encoder_layer_forward(src,                                  # type: Tensor
```

Comment:
Add to the docs that this is an unstable API, because we'll need to add support for slightly different standard transformers (e.g. BERT) and this is based on "All you need is attention" (which we should also cite and reference).